whole-body-controller-core/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
That file should not be pushed, it is an Eclipse configuration file. If it shows in staged changes we need to update the `.gitignore` file or add one if missing